shared/style/action_menu.css

```diff
@@ -268,9 +268,20 @@ html[dir="rtl"] [role="dialog"][data-type="action"] > header > a {
   font-style: italic;
 }
 
-[role="dialog"][data-type="action"] > menu > button:last-child {
+[role="dialog"][data-type="action"] > menu > .button-container:last-child {
+  position: fixed;
+  width: 100%;
+  background-color: #4d4d4d;
+  height: 7rem;
+  offset-inline-start: 0;
+  bottom: 0;
+}
+
+[role="dialog"][data-type="action"] > menu > .button-container:last-child button {
+  width: calc(100% - 3rem);
   position: fixed;
   bottom: 0;
+  margin: 0 1.5rem 1rem;
```

Comment:
I see a margin-bottom below that you can integrate here as well ?

I think this could be easier if the button-container would be "display: flex" and let the flexible layout do its magic to center the button... But maybe this is more than what you want to do here.